src/ralph/lib/table/table.py

```diff
@@ -0,0 +1,110 @@
+try:
+    from dj.choices import Choices
+    use_choices = True
+except ImportError:
+    Choices = None
+    use_choices = False
+
+from django.core.exceptions import FieldDoesNotExist
+from django.forms.utils import flatatt
+
+from ralph.admin.helpers import get_field_by_relation_path
+
+
+class TableRenderer(object):
```

Comment:
It's not renderer anymore (it isn't rendering anything) - think about better name